packages/react-storage/src/components/StorageBrowser/components/Table.tsx

```diff
@@ -29,50 +28,48 @@ interface TableProps {
 
 export const Table = ({ headers, rows }: TableProps): React.JSX.Element => {
   return (
-    <ViewElement className={`${STORAGE_BROWSER_BLOCK}__data-table`}>
-      <TableElement className={`${STORAGE_BROWSER_BLOCK}__table`}>
-        <TableHeadElement className={`${STORAGE_BROWSER_BLOCK}__table-head`}>
-          {headers.length ? (
-            <TableRowElement className={`${STORAGE_BROWSER_BLOCK}__table-row`}>
-              {headers.map(({ key, content }) => (
```

Comment:
This change is due to the View moving into the data table composable. No inner content was changed